gammapy/obs/datastore.py

```diff
@@ -278,3 +278,95 @@ def check_available_event_lists(self, logger=None):
                                    ''.format(obs_id, filename))
 
         return file_available
+
+
+def convert_obs_list_format_to_gammapy(obs_list, scheme):
+    """Convert oservation list from supported formats to Gammapy format.
+
+    This script calls the corresponding format converter, depending
+    on the value of the **scheme** parameter, in order to make
+    observation lists from different experiments comply to the format
+    described in :ref:`dataformats_observation_lists`.
+
+    Curretly only the H.E.S.S. scheme is supported, which follows the
+    guidelines dictated for CTA.
```

Comment:
There are no guidelines for CTA yet ... remove that statement (or the whole sentence).